### PR TITLE
EWPP-394: Project content type has wrong styling

### PR DESCRIPTION
**Pull request by enriquelacoma**
### Description

Include missing components/description_list in the sass style files.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "~1.5",
         "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/oe_content": "~1.11",
+        "openeuropa/oe_content": "dev-EWPP-0000",
```
Comment (nagyad) [inline composer.json]: This can be changed to 1.11 and add composer patch to master.

Comment (ademarco) [reply composer.json]: mh no let's use dev-master or else it will apply the patch on ewcms.

Comment (nagyad) [reply composer.json]: ok

Comment (enriquelacoma) [reply composer.json]: Updated

**Review by sergepavle — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**